src/Database/Console/Migrations/FreshCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Hyn\Tenancy\Database\Console\Migrations;
+
+use http\Exception\RuntimeException;
+use Hyn\Tenancy\Exceptions\ConnectionException;
+use Hyn\Tenancy\Traits\MutatesMigrationCommands;
+use Illuminate\Database\Console\Migrations\FreshCommand as BaseCommand;
+
+class FreshCommand extends BaseCommand
+{
+    use MutatesMigrationCommands;
+
+    /**
+     * Execute the console command
+     */
+    public function handle()
+    {
+        if (!$this->confirmToProceed()) {
+            return;
+        }
+
+        $this->input->setOption('force', $force = true);
+        $website_id = $this->option('website_id');
+        $realpath = $this->option('realpath');
+        $path = $this->input->getOption('path');
+
+        $website = $this->websites->query()->where('id', $website_id)->firstOrFail();
```

Comment:
Sorry to be a bother, this could be shortened to:

```php
$website = $this->websites->query()->findOrFail($website_id);
```

Reply:
Yeah, it can be! Thanks for the suggestion.